testing/src/main/java/org/conscrypt/java/security/AlgorithmParameterAsymmetricHelper.java

```diff
@@ -47,10 +47,10 @@ public void test(AlgorithmParameters parameters) throws Exception {
 
         Cipher cipher = Cipher.getInstance(algorithmName);
         cipher.init(Cipher.ENCRYPT_MODE, keyPair.getPublic(), parameters);
-        byte[] bs = cipher.doFinal(plainData.getBytes());
+        byte[] bs = cipher.doFinal(plainData.getBytes("UTF-8"));
```

Comment:
nit: I think we may have defined this somewhere in the test code already ... might be worth poking around.